app/src/main/java/com/cornellappdev/android/eatery/LoginFragment.java

```diff
@@ -0,0 +1,188 @@
+package com.cornellappdev.android.eatery;
+
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentTransaction;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+import android.widget.Button;
+import android.widget.CheckBox;
+import android.widget.CompoundButton;
+import android.widget.EditText;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+
+import com.cornellappdev.android.eatery.model.BrbInfoModel;
+import com.cornellappdev.android.eatery.network.GetLoginUtilities;
+import com.cornellappdev.android.eatery.network.JsonUtilities;
+import com.cornellappdev.android.eatery.presenter.AccountPresenter;
+
+public class LoginFragment extends Fragment {
+    private TextView mDescriptionText;
+    private Button mLoginButton;
+    private EditText mNetID;
+    private EditText mPassword;
+    private TextView mPrivacy;
+    private ProgressBar mProgressBar;
+    private CheckBox mSaveInfoCheck;
+    private AccountInfoFragment accountInfoFragment = new AccountInfoFragment();
+
+    private AccountPresenter mAccountPresenter = new AccountPresenter();
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+
+        View rootView = inflater.inflate(R.layout.fragment_login, container, false);
+        getActivity().setTitle("Login");
+
+       if(mAccountPresenter.isLoggedIn()){
```

Comment:
add a space - `if (...)`

Reply:
Figured out I didn't check the 'all places' scope that's why it didn't work. Fixed it!